man/systemd.network.xml

```diff
@@ -3001,6 +3001,15 @@ Token=prefixstable:2002:da8:1::</programlisting></para>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>UsePREF64=</varname></term>
```

Comment:
BTW, I think the flag is not necessary, at least now, as currently the parsed option is used only when we dump json.

Reply:
Yeah but later on we need it. No harm adding it